test/core/runtime/wasm_memory_test.cpp

```diff
@@ -12,10 +12,7 @@ using kagome::runtime::binaryen::kInitialMemorySize;
 using kagome::runtime::binaryen::roundUpAlign;
 using kagome::runtime::binaryen::WasmMemoryImpl;
 
-class MemoryHeapTest : public ::testing::Test, public WasmMemoryImpl {
- public:
-  MemoryHeapTest() : WasmMemoryImpl(getNewShellExternalInterface()) {}
-
+class MemoryHeapTest : public ::testing::Test {
```

Comment:
This file looks like reverted. I guess it wrong

Reply:
You are absolutely right. I will merge it correctly.